src/select/styled-components.js

```diff
@@ -152,3 +152,7 @@ export const SelectSpinner = styled('div', () => {
     paddingLeft: '50%',
   };
 });
+
+export const SelectionList = styled('div', props => {
```

Comment:
How about we call it `SelectionContainer`? Since it's not a list.

Reply:
ok